editor/project_manager.cpp

```diff
@@ -883,6 +966,11 @@ void ProjectManager::_load_recent_projects() {
 	tabs->set_current_tab(0);
 }
 
+void ProjectManager::_on_project_renamed() {
+	print_line("PRoject renamed");
```

Comment:
Extra capital letter here.

Reply:
Yeah and that should not been uploaded. I'm fixing it right away